tests/test_docs/test_bash_yaml/md_files/bash-erc1155-skills.md

```diff
@@ -86,9 +104,13 @@ default_routing:
   fetchai/oef_search:0.10.0: fetchai/soef:0.13.0
 ```
 ``` yaml
+---
+public_id: fetchai/p2p_libp2p:0.12.0
+type: connection
 config:
   delegate_uri: 127.0.0.1:11001
-  entry_peers: ['SOME_ADDRESS']
+  entry_peers:
+  - SOME_ADDRESS
```

Comment:
Please add multi address command here too

Reply:
we decided to retrieve it in a separated command, don't we? See above.